### PR TITLE
2000 furusawa

### PR DESCRIPTION
**Pull request by fkalholm**
I would like to merge the Furusawa-sims into the master branch. Please check all looks ok.

### CODE REVIEW

File: 2000_Furusawa/src/template/mat.dat [hunk 1]
```diff
@@ -1,33 +1,24 @@
 MEDIUM 1
```
Comment (nbassler) [inline 2000_Furusawa/src/template/mat.dat:1]: You shoudn't remove the Czub directory with this PR... Can you add the files back?

File: 2000_Furusawa/data/output/wdir/12C_V79_117/geo.dat [hunk 2]
```diff
@@ -0,0 +1,32 @@
+*---><---><--------><------------------------------------------------>
```
Comment (nbassler) [inline 2000_Furusawa/data/output/wdir/12C_V79_117/geo.dat]: ...also you shoud not commit generated files.

**Review by nbassler — CHANGES_REQUESTED** [review]

File: 2008_Czub/12C_33.2MeV/beam.dat [hunk 3]
```diff
@@ -0,0 +1,13 @@
+*         Input file FOR023.DAT for the SHIELD Transport Code
```
Comment (grzanka) [inline 2008_Czub/12C_33.2MeV/beam.dat]: Can you remove first two lines (some old comment and empty line) ? There are meaningless for this simulation

File: 2008_Czub/12C_33.2MeV/beam.dat [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+RNDSEED      	89736501     ! Random seed
+JPART0       	25           ! Incident particle type
+HIPROJ     	    12.0    6.0  ! A and Z of heavy ion
+TMAX0      	     4.0   0.0  ! Incident energy; (MeV/nucl)
+NSTAT           100   3000  ! NSTAT, Step of saving
+BEAMSIGMA       -5      -5   ! beam width
```
Comment (grzanka) [inline 2008_Czub/12C_33.2MeV/beam.dat]: Can you add comment what type of beam it is (flat square?)

File: 2000_Furusawa/data/output/wdir/12C_V79_102/beam.dat [hunk 5]
```diff
@@ -0,0 +1,13 @@
+*         Input file FOR023.DAT for the SHIELD Transport Code
```
Comment (grzanka) [inline 2000_Furusawa/data/output/wdir/12C_V79_102/beam.dat]: Can you remove first two lines ? They are meaningless here

File: 2000_Furusawa/data/output/wdir/12C_V79_102/beam.dat [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+RNDSEED      	89736501     ! Random seed
+JPART0       	25                 ! Incident particle type
+HIPROJ		12     6   
+TMAX0      	135     0.0  ! Incident energy; (MeV/nucl)
+NSTAT           10000000	50000         ! NSTAT, Step of saving
+BEAMSIGMA       -10         -10           ! beam width [cm]
```
Comment (grzanka) [inline 2000_Furusawa/data/output/wdir/12C_V79_102/beam.dat]: Can you add a comment specifying beam source type ? (flat square ?)

File: 2000_Furusawa/data/output/wdir/12C_V79_102/detect.dat [hunk 7]
```diff
@@ -0,0 +1,31 @@
+# Very simple test of new scoring system
```
Comment (grzanka) [inline 2000_Furusawa/data/output/wdir/12C_V79_102/detect.dat]: Can you remove first two line (especially first one)?

File: 2000_Furusawa/data/output/wdir/12C_V79_102/detect.dat [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    Y  -2.0  2.0    1
+    Z   5.0  5.01   1
+
+Filter
+    Name PrimPart
+    Z = 6   
+    A = 12  
+    GEN = 0
+
+Output
```
Comment (grzanka) [inline 2000_Furusawa/data/output/wdir/12C_V79_102/detect.dat]: Can you align Quantity lines with Geo line ?

File: 2000_Furusawa/data/output/wdir/12C_V79_117/beam.dat [hunk 9]
```diff
@@ -0,0 +1,13 @@
+*         Input file FOR023.DAT for the SHIELD Transport Code
```
Comment (grzanka) [inline 2000_Furusawa/data/output/wdir/12C_V79_117/beam.dat]: Same comments as to previous beam.dat files

File: 2000_Furusawa/data/output/wdir/12C_V79_117/detect.dat [hunk 10]
```diff
@@ -0,0 +1,31 @@
+# Very simple test of new scoring system
```
Comment (grzanka) [inline 2000_Furusawa/data/output/wdir/12C_V79_117/detect.dat]: Same comments as to previous geo.dat files

File: 2000_Furusawa/data/output/wdir/12C_V79_117/geo.dat (hunk 2 above)
Comment (grzanka) [reply 2000_Furusawa/data/output/wdir/12C_V79_117/geo.dat]: I agree

**Review by grzanka — CHANGES_REQUESTED** [review]

**Comment by fkalholm** [comment]
Moved data over as issues, and cleaned up the files

**Review by grzanka — APPROVED** [review]
Looks OK now

**Review by nbassler — APPROVED** [review]
This PR is too large for me to properly review it, but lets just approve, and then fix any mistakes from then on.

**Pull request merged**